template.yaml

```diff
@@ -20,5 +20,104 @@ Globals:
 
 Parameters:
   DataCiteMdsConfigs:
+    Type: String
     Description: Institution configs for DataCite MDS API
-    Default: '{{resolve:secretsmanager:dataCiteMdsConfigs:SecretString}}'
+    Default: '{{resolve:secretsmanager:dataciteMdsConfigs:SecretString}}'
+  DataCiteHost:
+    Type: String
+    Default: mds.test.datacite.org
+  DataCitePort:
+    Type: String
+    Default: 443
+  EventBusName:
+    Type : 'AWS::SSM::Parameter::Value<String>'
+    Default: nvaEventBusName
+  EventBusArn:
+    Type : 'AWS::SSM::Parameter::Value<String>'
+    Default: nvaEventBusArn
+  DlqArn:
+    Type : 'AWS::SSM::Parameter::Value<String>'
+    Default: nvaDlqArn
```

Comment:
TODO: Remove and replace with dedicated DLQ for handler as we want to know the source of which handler failed.

Reply:
Fixed dedicated DLQs for each handler, as we want to know the origin. See https://github.com/BIBSYSDEV/nva-publication-api/pull/97#discussion_r528626374